src/assets/cru-scss/_global-website-list.scss

```diff
@@ -1,4 +1,23 @@
+/***** TESTING *****/
```

Comment:
Does this still need to be here? There's also a lot of whitespace below this class...